src/client/redux/recipes/operations.js

```diff
@@ -6,28 +6,31 @@ const fetchRecipes = createAsyncThunk(
   async (_, thunkAPI) => {
     try {
       const response = await axios.get('/recipes');
-      return response.data.file;
+      return response.data;
```

Comment:
our response in api is `data.data.file` because we are return `json` with field `data.file` where we save our response.

Reply:
So should ale responses be done as that?